examples/nodejs/aws/eventbridge/webapp/src/svgs/corner-left-down.tsx

```diff
@@ -0,0 +1,9 @@
+export const CornerLeftDown = () => {
+  return (
+    <svg className="h-8 w-8 mt-2 text-gray-500" width="24" height="24" viewBox="0 0 24 24" strokeWidth="2"
+         stroke="currentColor" fill="none" strokeLinecap="round" strokeLinejoin="round">
+      <path stroke="none" d="M0 0h24v24H0z"/>
+      <path d="M18 6h-6a3 3 0 0 0 -3 3v10l-5 -5m10 0l-5 5"/>
+    </svg>
+  );
+};
```

Comment:
do you think we could add two EventBridge boxes to our "arch diagram"? so like, the arrows that come out of the ddb box would go to EventBridge boxes, and then those would have arrows that go to the momento boxes?

i think this would all be horizontal real estate, no vertical, so hopefully it should work?